docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -7,46 +7,43 @@ markdown: kramdown
 ---
 
 # IBM FHIR Server - Search Configuration Overview
-The [FHIR Specification](https://www.hl7.org/fhir/r4/search.html) defines a set of searchable fields for each resource type, and corresponding RESTful API.  The IBM FHIR Server supports searching using the specification defined search parameters and using tenant-specific search parameters (extensions).
+The [FHIR Specification](https://www.hl7.org/fhir/r4/search.html) defines a set of searchable fields for each resource type. The IBM FHIR Server supports searching via both specification-defined search parameters and tenant-specific search parameters.
 
 Specifically, the IBM FHIR Server supports searching on additional fields, including:
-* fields that are defined in the base specification, which are not configured for search;  
-* extension elements that a tenant may add to a standard FHIR resource type; and  
-* attributes that you define as part of a custom resource type  
+* fields that are defined in the base specification, which are not configured for search; and
+* extension elements that a tenant may add to a standard FHIR resource type
 
-The IBM FHIR Server allows deployers to define search parameters on a tenant-specific basis. This allows each tenant to share an instance of the FHIR server while maintaining the ability to have their own set of search parameters.
+The IBM FHIR Server allows deployers to define search parameters on a tenant-specific basis. This allows each tenant to share an instance of the FHIR server while maintaining the ability to have their own set of search parameters. Additionally, specificatin-defined search parameters can be filtered out in order to avoid the cost of extracting and storing the corresponding indices.
```

Comment:
```suggestion
The IBM FHIR Server allows deployers to define search parameters on a tenant-specific basis. This allows each tenant to share an instance of the FHIR server while maintaining the ability to have their own set of search parameters. Additionally, specification-defined search parameters can be filtered out in order to avoid the cost of extracting and storing the corresponding indices.
```